fwoa-tools/src/binaryConverter.unit.test.ts

```diff
@@ -0,0 +1,267 @@
+      const binaryResource: any = {
+        resourceType: 'Binary',
+        contentType: 'image/jpeg',
+        id: 'testBinaryObj',
+        meta: {
+          tag: [],
+          versionId: 1
+        }
+      };
+      AWSMock.mock(
+        'S3',
+        'listObjectsV2',
+        // eslint-disable-next-line @typescript-eslint/ban-types
+        (params: ListObjectsV2Request, callback: Function) => {
+          callback(null, {
+            Contents: [{ Key: 'testBinaryObj_1.png' }],
+            $response: {}
+          });
+        }
+      );
+
+      AWSMock.mock(
+        'S3',
+        'getObject',
+        // eslint-disable-next-line @typescript-eslint/ban-types
+        (params: GetObjectRequest, callback: Function) => {
+          callback(null, { Body: JSON.stringify(binaryResource), $response: {} });
+        }
+      );
+      AWSMock.mock(
+        'S3',
+        'upload',
+        // eslint-disable-next-line @typescript-eslint/ban-types
+        (params: PutObjectRequest, callback: Function) => {
+          // check that we are uploading the right binary resource
+          expect(params.Key).toBe('binaryUnitTests/Binary_converted_testBinaryObj/Binary-1.ndjson');
+          callback(null, { Body: JSON.stringify(binaryResource), $response: {} });
+        }
+      );
+      await convertBinaryResource(logs, fakeFile);
+    });
+
+    test('multi-tenancy = true', async () => {
+      process.env.MIGRATION_TENANT_ID = 'unitTestTenant';
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const binaryResource: any = {
+        resourceType: 'Binary',
+        contentType: 'image/jpeg',
+        id: 'testBinaryObj',
+        meta: {
+          tag: [],
+          versionId: 1
+        }
+      };
+      AWSMock.mock(
+        'S3',
+        'listObjectsV2',
+        // eslint-disable-next-line @typescript-eslint/ban-types
+        (params: ListObjectsV2Request, callback: Function) => {
+          callback(null, {
+            Contents: [{ Key: 'testBinaryObj_1.png' }],
+            $response: {}
+          });
+        }
+      );
+
+      AWSMock.mock(
+        'S3',
+        'getObject',
+        // eslint-disable-next-line @typescript-eslint/ban-types
+        (params: GetObjectRequest, callback: Function) => {
+          callback(null, { Body: JSON.stringify(binaryResource), $response: {} });
+        }
+      );
+      AWSMock.mock(
+        'S3',
+        'upload',
+        // eslint-disable-next-line @typescript-eslint/ban-types
+        (params: PutObjectRequest, callback: Function) => {
+          // check that we are uploading the right binary resource
+          expect(params.Key).toBe(
+            'unitTestTenant/binaryUnitTests/Binary_converted_testBinaryObj/Binary-1.ndjson'
+          );
+          callback(null, { Body: JSON.stringify(binaryResource), $response: {} });
+        }
+      );
+      await convertBinaryResource(logs, fakeFile);
```

Comment:
Should we add an expect for this? Something like this
`await expect(getFhirClient()).resolves.toEqual(expect.anything());`

Reply:
I can add this in yeah, I was mainly checking the expects in the mocks parameters but it makes sense to check that it resolves successfully.